apps/src/code-studio/pd/workshop_dashboard/components/survey_results/survey_rollup_table.story.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import SurveyRollupTable from '../../components/survey_results/survey_rollup_table';
+import reactBootstrapStoryDecorator from '../../../reactBootstrapStoryDecorator';
+
+const facilitator_rollups = {
+  facilitators: {
+    '1124': 'Facilitator Person 1',
+    '1125': 'Facilitator Person 2'
+  },
+  questions: {
+    facilitator_effectiveness_1494688268261028618:
```

Comment:
One important feature of a .story.jsx is that somebody can view it in storybook to understand how to use that component in their own code.  I imagine this set of magic numbers came from real data, but they are rather opaque and overwhelming here.  We should make sure that a .story.jsx is quite human-readable, so I'd encourage simplifying all of these numbers here.

Reply:
That's correct, they come from real data. I'll simplify them to make it easier to understand.